sympy/utilities/lambdify.py

```diff
@@ -50,9 +50,9 @@
     "E": "e",
     "I": "j",
     "ln": "log",
-    #"lowergamma":"lower_gamma",
+    "lowergamma":"gammainc",
```

Comment:
no, they both have the same keyword in mpmath, but have different parameters i.e. for upper gamma its gammainc(z,a,inf) and for lower gamma its gammainc(z,0,b)